core/src/main/java/org/apache/calcite/rel/rel2sql/RelToSqlUtils.java

```diff
@@ -66,7 +68,21 @@ private boolean isOperandAnalyticalInFollowingProject(RelNode rel, Integer rexOp
     return false;
   }
 
-  /** Returns whether an Analytical Function is present in filter condition. */
+  private boolean checkForRexCall(RexNode rexNode) {
+    if (rexNode instanceof RexCall) {
+      List<RexNode> listOfRexNode = ((RexCall) rexNode).getOperands();
+      for (RexNode node : listOfRexNode) {
+        if (node instanceof RexOver) {
```

Comment:
Unable to create a JAR for the calcite-core module. Please check it.

Reply:
Done